app/src/ui/add-repository/create-repository.tsx

```diff
@@ -265,8 +283,25 @@ export class CreateRepository extends React.Component<ICreateRepositoryProps, IC
     )
   }
 
+  private renderInvalidPathWarning() {
+    if (this.state.isValidPath) {
+      return null
+    }
+
+    return (
+      <Row className='warning-helper-text'>
+        <Octicon symbol={OcticonSymbol.alert} />
+        <p>
+          Cannot create repository at selected path. <br/>
```

Comment:
Not sure I can make this less wordy, but what about something like this:

> Directory could not be created at this path.
> Your user account may not have permissions to do this.

Or:

> Directory could not be created at this path.
> You may not have permissions to create a directory here.